doc/nrf/ug_thread_configuring.rst

```diff
@@ -467,8 +464,8 @@ The :kconfig:`CONFIG_OPENTHREAD_BUILD_OUTPUT_STRIPPED` Kconfig option will be di
 Updating libraries to debug version
 -----------------------------------
 
-You can also install only the debug version of the current OpenThread libraries (from Zephyr).
-This can be useful when debugging, but will take a significant amount of memory of the PC storage - it should be taken into account if one intends to commit those libraries to the repository.
+You can also install the debug version of the current OpenThread libraries (from Zephyr).
+This can be useful when debugging, but will take a significant amount of storage memory. Take this into account while committing those libraries to the repository.
```

Comment:
```suggestion
This can be useful when debugging, but will take a significant amount of storage memory.
Take this into account while committing those libraries to the repository.
```

Reply:
(new line for new sentence)